test/e2e/apps/job.go

```diff
@@ -18,11 +18,14 @@ package apps
 
 import (
 	"fmt"
+	batchv1 "k8s.io/api/batch/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
```

Comment:
This import line block seems for general libraries.
It is better to move the above lines to the import block of line 25 for k8s.io.

Reply:
Thank you. before I thought running hack/update-gofmt.sh would work. i will update it